yarn-project/aztec-faucet/terraform/variables.tf

```diff
@@ -2,23 +2,12 @@ variable "DEPLOY_TAG" {
   type = string
 }
 
-variable "RPC_URL" {
-  type    = string
-  default = "testnet"
-}
-
 variable "API_KEY" {
   type = string
 }
 
-variable "API_PREFIX" {
-  type    = string
-  default = ""
-}
-
 variable "CHAIN_ID" {
   type    = string
-  default = 31337
```

Comment:
with current setup this will throw an error as it's not being set anywhere, even if there is a default replacement inside the faucet code

Reply:
Can we set it explicitly somewhere? Defaults like that can be a bit of a footgun.

Reply:
Yep can set it as an env var